worlds/gen.world.jinja

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" ?>
+<!--
+Generated with jinja_world_gen.py
+@author: Benjamin Perseghetti
```

Comment:
I would be careful with this: you do are the author of the script, but you are not the author of the resulting models. Doesn't make sense to add you as an author to the generated worlds, specially when they include models you didn't create.

Reply:
I was assuming most would easily read that as being where the resultant worlds were generated from: jinja_world_gen.py using template: gen.world.jinja and the person they could reach out to if they have questions about generation. Considering that each world is user specified now and unique, I guess the only author of note would be the python path to the locally installed jinja2? Furthermore, it simply includes existing models, doesn't rewrite them, those model files obviously have their own authors (which are not being changed)... So I'm struggling to see the point.

Reply:
When I see an author tag I don't look at it as someone taking credit as much as someone willing to take responsibility for questions and issues. If people add to it and want to add themselves as an author or contributor they can as well (assuming they also desire to support it). I have made plenty of files and scripts that I have not included my contact info on because I had no desire to be bothered about them, in this case I want to provide support for it and that is the only reason why I have included it.

Reply:
>I have not included my contact info on because I had no desire to be bothered about them, in this case I want to provide support for it and that is the only reason why I have included it.

If you want to provide support, put your contact information in the code generator. Generated code shouldn't have licensing or contacts.

Reply:
Is gen.world.jinja somehow not part of the generator?